From 11093e6f0de9783c26355e0962f210a7ae18ab3c Mon Sep 17 00:00:00 2001
From: Bagas Sanjaya <bagasdotme@gmail.com>
Date: Sat, 2 Jul 2022 11:23:50 +0700
Subject: [PATCH] Documentation: dm writecache: Render status list as list
Git-commit: 11093e6f0de9783c26355e0962f210a7ae18ab3c
Patch-mainline: v6.0-rc1
References: git-fixes

The status list isn't rendered as list, but rather as normal paragraph,
because there is missing blank line between "Status:" line and the list.

Fix the issue by adding the blank line separator.

(Coly Li: rebased for Linux v4.12 based SUSE kernel)

Fixes: 48debafe4f2fea ("dm: add writecache target")
Signed-off-by: Bagas Sanjaya <bagasdotme@gmail.com>
Signed-off-by: Mike Snitzer <snitzer@kernel.org>
Signed-off-by: Coly Li <colyli@suse.de>

---
 Documentation/device-mapper/writecache.txt |    1 +
 1 file changed, 1 insertion(+)

--- a/Documentation/device-mapper/writecache.txt
+++ b/Documentation/device-mapper/writecache.txt
@@ -47,6 +47,7 @@ Constructor parameters:
 		  with nofua. The user should test it
 
 Status:
+
 1. error indicator - 0 if there was no error, otherwise error number
 2. the number of blocks
 3. the number of free blocks
